core/src/ons/platform.js

```diff
@@ -138,14 +138,17 @@ class Platform {
   //----------------
   /**
    * @method isIOS
-   * @signature isIOS()
+   * @signature isIOS([ignoreSelectedPlatform])
+   * @param {Boolean} ignoreSelectedPlatform
+   *   [en]If true, selected platform is ignored and the actual platform is returned.[/en]
+   *   [ja][/ja]
    * @description
-   *   [en]Returns whether the OS is iOS.[/en]
+   *   [en]Returns whether the OS is iOS. By default does not ignore selected platform.[/en]
```

Comment:
I would rephrase this change to not use the word ignore, more like `By default, will return manually selected platform if it is set`